data/teamData.js

```diff
@@ -1,69 +1,75 @@
 export default {
   directors: [
     {
-      name: "Timothy Ko",
+      name: "Angad Garg",
       year: "'20",
       quote:
-        "I joined Hack4Impact looking to work with incredible and hard-working students while using my skills to do something more meaningful. With Hack4Impact, I’d like to give this same opportunity to our members while pushing them to learn and grow tremendously both personally and professionally.",
-      linkedin: "https://linkedin.com/in/timothy-ko"
+        "Hack4Impact has given me the opportunity to apply my skills towards important causes while improving myself at the same time. I hope to help others experience this personal growth, see the impact their work can have, and collaborate with other people who care just as much. ",
+      linkedin: "https://www.linkedin.com/in/angad-garg/"
     },
     {
-      name: "Angad Garg",
+      name: "Roy Chiu",
       year: "'20",
       quote:
-        "Hack4Impact has given me the opportunity to apply my skills towards important causes while improving myself at the same time. I hope to help others experience this personal growth, see the impact their work can have, and collaborate with other people who care just as much. ",
-      linkedin: "https://www.linkedin.com/in/angad-garg/"
-    }
+        "I joined Hack4Impact because I wanted to work on impactful products with like-minded, dedicated, and talented people. I hope to give others the same opportunities to work on projects they love with people they love through Hack4Impact.",
+      linkedin: "https://www.linkedin.com/in/royguanchiu/"
+    },
   ],
   team: [
     {
-      name: "Annie Wu",
-      role: "Director of PR",
-      linkedin: "https://www.linkedin.com/in/anniewu701/"
+      name: "Hana Rimawi",
+      role: "Community Director",
+      linkedin: "https://www.linkedin.com/in/hana-rimawi/"
     },
     {
-      name: "Doris Zhao",
-      role: "Community Director"
+      name: "Daniel Choi",
+      role: "Director of ER",
```

Comment:
I think this should just be Director of External Relations or just simply External Relations since Community Director is just the title so we should be consistent. Also not many people may know that ER is external relations since isn't it usually PR/public relations

Reply:
yea I'll see if it fits well

Reply:
rolls over into 2 lines, but still looks fine.